### PR TITLE
Flow fix me cleanup 3

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: packages/relay/index.js [hunk 1]
```diff
@@ -7,14 +7,15 @@ import {
...
+  type Environment,
 } from '@kiwicom/relay';
 import { Alert } from '@kiwicom/mobile-localization';
 
-import PublicEnvironment from './src/PublicEnvironment';
-import PrivateEnvironment from './src/PrivateEnvironment';
 import ConnectionManager from './src/ConnectionManager';
 import type { QueryRendererProps as _QueryRendererProps } from './types';
 
+export { default as PublicEnvironment } from './src/PublicEnvironment';
```
Comment (RobinCsl) [inline packages/relay/index.js]: So we don't really need the PrivateEnvironment?

Comment (tbergq) [reply packages/relay/index.js]: We don't need to export at this point at least

**Comment by tbergq** [comment]
> Could you update

Yes, I will do it later today. I didn't even remember that we had these docs anymore 😂

**Comment by tbergq** [comment]
@RobinCsl 🚀

**Pull request merged**